api/valor_api/backend/core/dataset.py

```diff
@@ -165,9 +165,12 @@ def get_paginated_datasets(
             "Offset should be an int greater than or equal to zero. Limit should be an int greater than or equal to -1."
         )
 
-    datasets_subquery = (
-        Query(models.Dataset.id.label("id")).filter(filters).any()
-    )
+    advanced_filter = filters.to_advanced_filter() if filters else None
```

Comment:
the idea of an "advanced filter" is a bit confusing. can I assume this concept is still going away in a near-term PR?

Reply:
`AdvancedFilter` will assume the name `Filter` when we make the switch at the API / Client level.

Reply:
I am considering it advanced as it is a superset of the existing `Filter`